android/gradle.properties

```diff
@@ -1,5 +1,5 @@
 RNPDFEditor_kotlinVersion=1.7.0
-RNPDFEditor_minSdkVersion=21
+RNPDFEditor_minSdkVersion=26
 RNPDFEditor_targetSdkVersion=31
```

Comment:
Lets bump `RNPDFEditor_targetSdkVersion` & `RNPDFEditor_compileSdkVersion`  to `33`